app/assets/scripts/components/aa-table-vpromms.js

```diff
@@ -126,37 +121,30 @@ const AATable = React.createClass({
-          const vprommFieldInDB = (this.props.sources.indexOf(vpromm.id) !== -1);
-          const vprommInDB = (this.props.vpromms.indexOf(vpromm.id) !== -1);
-
-          let propBtnLabel = this.state.expandedId === vpromm.id ? 'Hide' : 'Show';
+          const vprommFieldInDB = (this.props.fieldIds.indexOf(vpromm) !== -1);
+          let propBtnLabel = this.state.expandedId === vpromm ? 'Hide' : 'Show';
           let propBtnClass = classnames('bttn-table-expand', {
-            'bttn-table-expand--show': this.state.expandedId !== vpromm.id,
-            'bttn-table-expand--hide': this.state.expandedId === vpromm.id
+            'bttn-table-expand--show': this.state.expandedId !== vpromm,
+            'bttn-table-expand--hide': this.state.expandedId === vpromm
           });
           let propContainerClass = classnames('table-properties', {
-            'table-properties--hidden': this.state.expandedId !== vpromm.id
+            'table-properties--hidden': this.state.expandedId !== vpromm
           });
-
+          const roadProps = this.props.propertiesData[vpromm];
           return (
-            <tr key={`vpromm-${vpromm.id}`} className={classnames({'alt': i % 2})}>
-              <th className={classnames({'added': vprommInDB, 'not-added': !vprommInDB})}>{ vprommInDB ? this.renderVProMMsLink(vpromm.id) : vpromm.id }</th>
-              <td className={classnames({'added': vpromm.inTheDatabase, 'not-added': !vpromm.inTheDatabase})}>{vpromm.inTheDatabase ? t('added') : t('not added')}</td>
-              <td className={classnames({'added': vprommFieldInDB, 'not-added': !vprommFieldInDB})}>{ vprommFieldInDB ? this.renderFieldMapButtons(vprommFieldInDB, vpromm.id) : ''}</td>
+            <tr key={vpromm} className={classnames({'alt': i % 2})}>
+              <th>{ this.renderVProMMsLink(vpromm) }</th>
+              <td className={classnames({'added': vprommFieldInDB, 'not-added': !vprommFieldInDB})}>{ vprommFieldInDB ? this.renderFieldMapButtons(vprommFieldInDB, vpromm) : ''}</td>
               <td className='table-properties-cell'>
-                <button type='button' className={propBtnClass} onClick={this.onPropertiesClick.bind(null, vpromm.id)}><span>{propBtnLabel}</span></button>
+                <button type='button' className={propBtnClass} onClick={this.onPropertiesClick.bind(null, vpromm)}><span>{propBtnLabel}</span></button>
                 <div className={propContainerClass}>
                   <dl className='table-properties-list'>
-                    <dt>Key 1</dt>
-                    <dd>Value 1</dd>
-                    <dt>Very_long_key_here_yes_it_is</dt>
-                    <dd>A_very_long_value_for_a_very_long_key</dd>
-                    <dt>Key 3</dt>
-                    <dd>Value 3</dd>
-                    <dt>Key 4</dt>
-                    <dd>Value 4</dd>
-                    <dt>Key 5</dt>
-                    <dd>Value 5</dd>
+                    <dt>Admin Level</dt>
```

Comment:
Context on the data: there are going to be a _lot_ of fields once we're given tabular DRVN data. So, I'd just show _all_ properties, and show the raw-uncleaned key and value. No need to specially label, and no need to standardize values.

Maybe the only thing to do would be to limit the precision of some of the numbers, so they don't take up too much room.